cmd/util/util.go

```diff
@@ -210,3 +215,74 @@ minikube config set WantKubectlDownloadMsg false
 			verb, fmt.Sprintf(installInstructions, constants.DefaultKubernetesVersion, goos, runtime.GOARCH))
 	}
 }
+
+type SX map[string]interface{}
+
+func StoreProcessToFile(p os.Process, filename string) error {
+	encoded := ToGOB64(map[string]interface{}{
+		filepath.Base(filename): p,
+	})
+	return ioutil.WriteFile(filename, []byte(encoded), 0644)
+}
+
+func ReadProcessFromFile(filename string) (os.Process, error) {
```

Comment:
Any reason we can't just use `os.FindProcess(pid int)` here and only write the pid?